src/Pages/Main/Components/StoreDetail/Info.js

```diff
@@ -63,7 +59,23 @@ const Info = props => {
     },
     {
       title: '인근 전철역',
-      content: <Content>{props.metro_stations.toString()}</Content>,
+      content: (
+        <Content>
+          {props.metro_stations.map((data, idx) => (
+            <ul key={idx}>
+              <Distance>
+                {(data.name + '(' + data.line + ')').toString()}까지 거리:{' '}
```

Comment:
가독성이 좋은 문법을 사용하면 더 좋을 것 같아요.
```suggestion
                {`${data.name}(${data.line})`}까지 거리:{' '}
```

Reply:
앗 네네 수정하겠습니다. 감ㅅ.ㅏ합니다